torch/fx/passes/infra/partitioner.py

```diff
@@ -140,7 +99,7 @@ def assign(node: Node, id: Optional[int] = None):
                 for j in range(i + 1, len(user_partitions_list)):
                     pi = user_partitions_list[i]
                     pj = user_partitions_list[j]
-                    dependency = self.__partition_depends_on(pi, pj)
+                    dependency = self.graph_viewer.partition_depends_on(pi.nodes, pj.nodes)
```

Comment:
for this specific case is it true that: for a two given inputs `pi.nodes` and `pj.nodes`, 1 and -1 cannot both be valid values of `dependency`?

Reply:
yes.